flink-rpc/flink-rpc-akka/pom.xml

```diff
@@ -39,7 +39,6 @@ under the License.
 	<properties>
 		<pekko.version>1.1.2</pekko.version>
 		<scala.binary.version>2.12</scala.binary.version>
-		<scala.version>2.12.16</scala.version>
```

Comment:
should we not have `<scala.version>2.12.20</scala.version>` here like the other pom files?

Reply:
what other pom files are you referring?
Here it was rewritten since it required 2.12.16 and higher
With this PR it inherits 2.12.20

Reply:
@snuyanzin the other pom files in the pr. If this inherits that is perfect.

Reply:
it does